### PR TITLE
Bug Fix #1323

### LINKED ISSUE #1323: ERROR:  type "double" does not exist for generated schema using createStatements
We use Slick 3.1 and it generate wrong create statements for Double PostgreSQL column, it generate DOUBLE instead of double precision !

    val seqTables = Seq(
      Tables.coffees.schema
      , Tables.suppliers.schema
    )

    val db = Database.forConfig("h2mem1")
  
    println("---create statements")
    seqTables.tail.fold(seqTables.head)(_ ++ _).createStatements
      .foreach(statement => println(statement.replace("DOUBLE", "double precision")+";"))

### PR DESCRIPTION
**Pull request by amritsinghbains**
https://github.com/slick/slick/issues/1323

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @amritsinghbains,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: slick/src/main/scala/slick/jdbc/PostgresProfile.scala [hunk 1]
```diff
@@ -71,6 +71,7 @@ trait PostgresProfile extends JdbcProfile {
         case (VarCharPattern(str),"String") => Some(Some(str))
         case (IntPattern(v),"Int") => Some(Some(v.toInt))
         case (IntPattern(v),"Long") => Some(Some(v.toLong))
+        case (IntPattern(v),"Double") => Some(Some(v.toDouble))
```
Comment (cvogt) [inline slick/src/main/scala/slick/jdbc/PostgresProfile.scala]: IntPattern sounds wrong. Shouldn't this allow for a decimal point? Also would be good to have a test for this. Not sure where the best place for the test is, but probably somewhere under https://github.com/slick/slick/tree/master/slick-testkit/src/codegen

**Comment by cvogt** [comment]
@amritsinghbains  did you have time to check my comment? we also need you to sign the cla

**Comment by cvogt** [comment]
This seem wrong. Closing this. Feel free to reopen, if I am wrong about this.

**Pull request closed without merging**